runtime/litentry/src/lib.rs

```diff
@@ -435,7 +435,7 @@ impl pallet_utility::Config for Runtime {
 	type Event = Event;
 	type Call = Call;
 	type PalletsOrigin = OriginCaller;
-	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
+	type WeightInfo = ();
```

Comment:
This change can be reverted now, as the benchmark files were updated, please be sure to correct all 4 runtimes when you make changes

Reply:
Yes. For the moment, only Litmus is supported. The rest three would be updated in further upgrade.